demo/src/main/java/uk/gov/homeoffice/digital/sas/demo/EntitiesApplication.java

```diff
@@ -2,8 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Import;
+import uk.gov.homeoffice.digital.sas.jparest.config.ApplicationConfig;
 
 @SpringBootApplication
+@Import(ApplicationConfig.class)
```

Comment:
Just wondering about the docs we were looking at before regarding the spring.factories file. 
I believe we concluded from the docs that the spring.factories file is no longer supported and the alternative is to use an 'imports file' mentioned in 2.7 release notes.
Would that not work and allows us to not have to add this import annotation on every client?
If you've concluded that doesn't work and this is the way forward then maybe we can just delete the spring.factories file too.

> https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#auto-configuration-files
> https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes#auto-configuration-registration

Reply:
Good point @daleelR . Actioned